src/Client.php

```diff
@@ -322,12 +322,13 @@ protected function isPipe($contents): bool
      * Create a new file with the contents provided in the request.
      *
      * Do not use this to upload a file larger than 150 MB. Instead, create an upload session with upload_session/start.
+     * For $mode you can use an object, for example: $mode=["update"=>$SomeRev, ".tag"=> "update"]
```

Comment:
Your example is array. So `$mode` can be an array too (and not an object)?

Reply:
Yes, I was confused but you are right So $mode can be an array too.